orchestrator/terraform/modules/heimdall/lambdas.tf

```diff
@@ -135,7 +135,7 @@ resource "aws_lambda_function" "repo-scan" {
   architectures = [var.lambda_architecture]
   timeout       = var.repo_scan_lambda_timeout
   memory_size   = 256
-  role          = aws_iam_role.lambda-assume-role.arn
```

Comment:
Shouldn't this role be removed? As well as the policy attachments for this role?

Reply:
Good catch